.github/workflows/main.yml

```diff
@@ -0,0 +1,48 @@
+name: tests
+
+on: [push, pull_request]
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+        - python-version: "3.6"
+          env:
+            TOXENV: py
+        - python-version: "3.7"
+          env:
+            TOXENV: py
+        - python-version: "3.8"
+          env:
+            TOXENV: py
+        - python-version: "3.9"
+          env:
+            TOXENV: py
+        - python-version: "3.10"
+          env:
+            TOXENV: py
+        - python-version: "3.7"
```

Comment:
I think @kmike’s recommendation for types is to test with the latest version of Python.

Reply:
Latest overall version of Python (3.11) or latest version used in tests (3.10)?

For now, I changed it to 3.10 in 6ec36ee02f3eee6700dd79a763f5b4bf5d41e94c